### ISSUE TITLE
Persona remove button doesn't have tooltip

### ISSUE THREAD

**Issue by ravi-405** [microsoft/fluentui#14902]
User Experience:
Users who depends on screen reader are facing difficulty as title is not defined for "People picker" remove button.

Repro Steps:
Step 1: Open https://fabricweb.z5.web.core.windows.net/oufr/7.105.2/#/examples/peoplepicker
Step 2: Search for any user in the people picker and select one user.
Step 3: Navigate to "Remove" button in persona and observe the tooltip.

Actual Result:
Close(X) button tooltip is not defined when focus is on close button.

Expected Result:
Close(X) button should have tooltip.

**Comment by paulgildea** [comment]
@ravi-405 Thanks for the feedback!

@joschect - Is there any way to target the (X) button on the People Picker to wrap with a `ToolTip`?

**Comment by miroslavstastny** [comment]
@ravi-405 - can you please specify what screenreader/browser combination have you used and also your key sequence to focus the **X** button?

FYI @smhigley, @kolaps33 - not sure what the expected remove item workflow is.

**Comment by ravi-405** [comment]
@miroslavstastny , Using NVDA + edge dev combination.

Key Sequence - on mouse hover.

Thanks

**Comment by v-ajrai** [comment]
@paulgildea .
I am also facing the similar issue. How much could be the ESA for this to e resolved.
![external_11](https://user-images.githubusercontent.com/62233982/98778630-a91d0b00-2418-11eb-9990-30883a4d9098.png)

**Comment by smhigley** [comment]
I don't think an accessibility tag is appropriate for this issue. Adding a tooltip has dubious accessibility benefits for an icon as well-known as close/clear, and definite accessibility drawbacks. Particularly within a control like a combobox, adding a secondary popup in addition to the main listbox and text input is going to cause complications around e.g. keyboard dismissal.

I wouldn't go so far as to say accessibility concerns mean that a tooltip *shouldn't* be added, but it's certainly not a WCAG requirement and debateably not a net accessibility benefit.

**Comment by joshmccall221** [comment]
You need to specify  `removeButtonAriaLabel`:
```
    <CompactPeoplePicker
        removeButtonAriaLabel={'Remove'} 
  ```
[see issue #14776:](https://github.com/microsoft/fluentui/issues/14776#issuecomment-768607215)
